Rakefile

```diff
@@ -158,7 +158,7 @@ end
 def update_readme_version(version)
   IO.write("README.md",
     File.open("README.md") do |file|
-      file.read.gsub(/:drop-in:\d+\.\d+\.\d+'/, ":drop-in:#{version}'")
+        file.read.gsub(/com.braintreepayments.api:drop-in:\d+\.\d+\.\d+(-.*)?'/, "com.braintreepayments.api:drop-in:#{version}'")
```

Comment:
It didn't look like the README drop-in version was updated in the last GA release (5.3.0), so I'm updating to the regex from the migration guide update task